packages/augur-ui/src/modules/market-cards/common.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+import { 
+  CATEGORICAL,
+  SCALAR,
+  YES_NO
+} from 'modules/common/constants';
+import { createBigNumber } from 'utils/create-big-number';
+
+import Styles from 'modules/market-cards/common.styles';
+
+export interface PercentProps {
+  percent: number;
+}
+
+export const Percent = (props: PercentProps) => (
+  <div className={Styles.Percent}>
+  	<span style={{width: props.percent + "%"}}></span>
+  </div>
+);
+
+export interface OutcomeProps {
+  name: string;
+  percent: number;
+  invalid?: Boolean;
+  index: number;
+}
+
+export const Outcome = (props: OutcomeProps) => (
+  <div className={classNames(Styles.Outcome, {[Styles.invalid]: props.invalid, [Styles[`Outcome-${props.index}`]]: !props.invalid})}>
+  	<div>
+    	<span>{props.name}</span>
+    	<span>{props.percent}%</span>
+    </div>
+    <Percent percent={props.percent} />
+  </div>
+);
+
+export interface ScalarOutcomeProps {
+  scalarDenomination: string;
+  min: number;
+  max: number;
+  percent: number;
+}
+
+function calculatePosition(min, max, percent) {
+	const range = createBigNumber(max).minus(createBigNumber(min));
+	const pricePercentage = createBigNumber(percent || 0)
+	  .minus(min)
+	  .dividedBy(range)
+	  .times(createBigNumber(100)).toNumber();
+
+	return percent === null
+	  ? 50
+	  : pricePercentage
+}
+
+export const ScalarOutcome = (props: ScalarOutcomeProps) => (
+  <div className={Styles.ScalarOutcome}>
+  	<div>
+  		<span style={{left: calculatePosition(props.min, props.max, props.percent) + '%'}}>{props.percent}</span>
```

Comment:
Dont use percent ‘%’ for scalar just the number. If percent which is actually last trade price is null then put’-‘

Reply:
this is for the style, how much to move the dot over